Edit four Access Controls pages for Cloud users (#11826)
gravitational/teleport: docs/pages/access-controls/getting-started.mdx

description: Get started using Teleport Access Controls.
---

> In Teleport, any local, SSO, or robot user can be a member of one or several

Reviewer: Not sure I love this working.

A user isn't really a "member" of a role.
A user _has_ one or more roles assigned to them.

roles. Roles govern access to databases, SSH servers, Kubernetes clusters,
Windows desktops, and web apps.

> We will start with local users and preset roles, map SSO users to roles, and

Reviewer: ```suggestion
We will start with local users and preset roles, assign roles to SSO users, and
```

## Step 1/3. Add local users with preset roles

Teleport provides several preset roles: `editor`, `auditor` and `access`.

> Members of the `editor` role can modify cluster configuration, members of

Reviewer: ```suggestion
Users with the `editor` role can modify cluster configuration, users with
```

Similar to above - I think it's best to describe roles as something assigned to users, instead of "groups" that you join or become a member of.

Reviewer 2: To further clarify roles vs. groups, just for my own understanding as well:

Roles are sets of rights. Groups are sets of people.

Roles can also be functions that one person or groups of people can be permitted to do.

          # map github admin team to Teleport's "access" role
          logins: ["access"]
    ```

    Create the `github` resource:

    ```code
    $ tctl create github.yaml
    ```

  </ScopedBlock>

  <ScopedBlock scope={["enterprise", "cloud"]}>

  Create a SAML or OIDC application that Teleport can integrate with, then
  create an authentication connector that maps users within your application to
  Teleport roles.

  <Tabs>
  <TabItem label="SAML">

  Follow our [SAML Okta Guide](../enterprise/sso/okta.mdx#configure-okta) to
  create a SAML application.

  Save the file below as `okta.yaml` and update the `acs` field.
  Any member in Okta group `okta-admin` will assume a built-in role `admin`.

>   ```yaml

Reviewer 3: Would it make sense to move SAML connector example into a separate file and include here, like it's done with oidc-connector.yaml below?

Author: I think that makes sense if we replace other `saml` connector examples with the partial. I'll create an issue since I think this is out of scope for this PR.

Author: https://github.com/gravitational/teleport/issues/11957
